.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
include only the .idea folder. Can remove the subfolder entries of .idea

Reply:
Done.